SimTracker/TrackTriggerAssociation/plugins/TTStubAssociator.h

```diff
@@ -53,11 +53,11 @@ class TTStubAssociator : public edm::stream::EDProducer<> {
 
 private:
   /// Data members
-  std::vector<edm::InputTag> TTStubsInputTags;
-  std::vector<edm::InputTag> TTClusterTruthInputTags;
+  std::vector<edm::InputTag> ttStubsInputTags;
```

Comment:
member variables should end in underscore

Reply:
done